docs/dg/dev/backend-development/data-manipulation/data-publishing/implement-publish-and-synchronization.md

```diff
@@ -87,70 +86,63 @@ For example, the following code defines events for publishing for the cases when
     public const ENTITY_SPY_GLOSSARY_TRANSLATION_DELETE = 'Entity.spy_glossary_translation.delete';
 ```
 
-The synchronization events are auto-generated and linked by the synchronization process. Therefore, no explicit event declaration is necessary for the default behavior.
-
 ## 3. Configure Publish & Synchronization queues
 
-Configure the publish queues where all publish events are exchanged. The default generic publish queue is `event`. You can define the queue in `data/shop/development/current/vendor/spryker/product/src/Spryker/Zed/Publisher/PublisherConfig::getPublishQueueName()`. Also, you can use separate publish queue per listener class (see [Listen to Publish Events](#listen-to-publish-events)).
-
-{% info_block infoBox "Creating a queue"%}
+Configure the publish queues where all publish events are exchanged. The default generic publish queue is not configured from the core, but all boilerplates have it as `publish` on the project level OOTB. You can re-define the default queue name in `PublisherConfig::getPublishQueueName()`.
+If you want to use different queue name for your entity you need to configure it by providing a key as a queue name in `PublisherDependencyProvider::getPublisherPlugins()` and also in the `QueueDependencyProvider::getProcessorMessagePlugins()` by setting message and sync processors.
```

Comment:
I would add an example here as it was done for glossaries event names and let's also mention the entire namespace not just the class name